styles/globals.css

```diff
@@ -72,11 +72,16 @@ h3 {
 }
 
 h3 button {
-  display: none;
+  will-change: opacity;
+
+  opacity: 0;
+  border-radius: 50%;
```

Comment:
Not sure about `border-radius: 50%;`
<img width="110" alt="image" src="https://user-images.githubusercontent.com/16056918/198996143-e279aa7a-43d8-4e47-a326-a1da71370b0e.png">

I would like to have consistent style across all similar elements (in this case buttons)

Reply:
Sure